sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -57,6 +57,7 @@ export class BatchingReceiver extends MessageReceiver {
     // Clears the token renewal timer. Closes the link and its session if they are open.
     await this._closeLink(this._receiver);
     this.detachedError = receiverError;
+    await super.onDetached(receiverError);
```

Comment:
Since we are now calling `super.onDetached()`, we don't need to call `this_closeLink()` anymore